src/chart.cpp

```diff
@@ -454,7 +454,7 @@ void Chart3D::pushTicktextCoords(float x, float y, float z)
 void Chart3D::generateChartData()
 {
     CheckGL("Begin Chart3D::generateChartData");
-    static const float border[] = { -1, -1, -1,  1, -1, -1,  -1, -1, -1,  -1, 1, -1,  -1, 1, -1,  -1, 1, 1 };
+    static const float border[] = { -1, -1, 1,  -1, -1, -1,  -1, -1, -1,  1, -1, -1,  1, -1, -1,  1, 1, -1 };
```

Comment:
Those are coordinates for the endpoints of the x, y, z, axes. To swap the x,y axes, I changed the order of rotations and therefore needed to change the endpoints of the drawn axes so they would draw correctly after the new rotations.